HW5/task_3.py

```diff
@@ -0,0 +1,16 @@
+file_path = "./test/data/text.txt"
+
+letter_counts = {}
+
+with open(file_path, "r") as file:
+    for line in file:
+        for letter in line:
+            if letter.isalpha():
+                letter = letter.lower()
+                if letter in letter_counts:
+                    letter_counts[letter] += 1
+                else:
+                    letter_counts[letter] = 1
```

Comment:
too complicated. 2 loops and 2 if blocks in one algorithm is not good. Please think about another solution